app/inc/dispatcher.php

```diff
@@ -21,7 +21,7 @@
         $page_descr = 'One source locale, get search results for two target locales';
         break;
     case 'news':
-        $view = 'changelog';
+        $controller = 'changelog';
         $page_title = 'Transvision News. Version Notes';
```

Comment:
Never really thought about this title. Maybe `Transvision News and Release Notes`?